src/otx/backend/openvino/models/detection.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Class definition for detection model entity used in OTX."""
+
+# type: ignore[override]
```

Comment:
can you elaborate why this is ignored?

Reply:
[Removed](https://github.com/open-edge-platform/training_extensions/pull/4374/commits/677f0d9e5138e65916f4b9c58ab08e398ffa41ae)